src/icon-loader.c

```diff
@@ -120,7 +120,20 @@ process_rel_name(struct icon_ctx *ctx, const char *icon_name,
 static int
 process_abs_name(struct icon_ctx *ctx, const char *icon_name)
 {
-	wlr_log(WLR_ERROR, "absolute paths not yet supported");
+	ctx->path = xstrdup(icon_name);
+	if (str_endswith(icon_name, "png")) {
```

Comment:
I feel like checking for ".png" (with a dot) would be slightly more robust.

Reply:
Good point. Done.